widgets/menu.js

```diff
@@ -0,0 +1,77 @@
+var React = require('react');
+var cx = require('classnames');
+
+var utils = require('../lib/widget-utils.js');
+
+var defaultTemplate = `<a href="{{href}}">{{name}}</a> {{count}}`;
```

Comment:
I would like the `{{href}}` to be given to every "refinement list" or "refinement link", so that you can generate real links and ctrl+click them or middle click them etc. Right now not implemented I agree. what do you think?